### PR TITLE
Add blog post welcoming yxxhero

### PR DESCRIPTION
**Pull request by karenhchu**
Signed-off-by: Karen Chu <karen.chu@microsoft.com>

### CODE REVIEW

File: content/en/blog/2022-11-14-welcome-yxxhero.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+title: "Helm welcomes yxxhero as our newest helm-www repo maintainer"
+slug: "welcome-yxxhero"
+authorname: "Karen Chu"
+author: "@karenhchu"
+authorlink: "https://twitter.com/karenhchu"
+date: "2022-11-14"
+---
+
+The Helm project is happy to welcome [yxxhero](https://github.com/yxxhero) as our newest maintainer for the helm-www repo! 
+<!--more--> yxxhero has been a long time contributor to the Helm project over the years. With a supermajority vote from the existing maintainers, yxxhero will be joining to help with Helm documentation. With their background, yxxhero will continue to help streghten our both simplified Chinese and traditional Chinese docs.
```
Comment (karenhchu) [inline content/en/blog/2022-11-14-welcome-yxxhero.md:11]:
```suggestion
<!--more--> yxxhero has been a long time contributor to the Helm project over the years. With a supermajority vote from the existing maintainers, yxxhero will be joining to help with Helm documentation. With their background, yxxhero will continue to help strengthen both our simplified Chinese and traditional Chinese docs.
```

Comment (karenhchu) [reply content/en/blog/2022-11-14-welcome-yxxhero.md:11]:
```suggestion
<!--more--> yxxhero has been a long time contributor to the Helm project over the years. With a supermajority vote from the existing maintainers, yxxhero will be joining to help with Helm documentation. With their background, yxxhero will continue to help streghten our both simplified and traditional Chinese docs.
```

**Review by karenhchu — COMMENTED** [review]
fixing a few grammar issues

File: content/en/blog/2022-11-14-welcome-yxxhero.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+title: "Helm welcomes yxxhero as our newest helm-www repo maintainer"
+slug: "welcome-yxxhero"
+authorname: "Karen Chu"
+author: "@karenhchu"
+authorlink: "https://twitter.com/karenhchu"
+date: "2022-11-14"
+---
+
+The Helm project is happy to welcome [yxxhero](https://github.com/yxxhero) as our newest maintainer for the helm-www repo! 
+<!--more--> yxxhero has been a long time contributor to the Helm project over the years. With a supermajority vote from the existing maintainers, yxxhero will be joining to help with Helm documentation. With their background, yxxhero will continue to help strengthen both our simplified Chinese and traditional Chinese docs.
```
Comment (karenhchu) [inline content/en/blog/2022-11-14-welcome-yxxhero.md:11]:
```suggestion
<!--more--> yxxhero has been a long time contributor to the Helm project over the years. With a supermajority vote from the existing maintainers, yxxhero will be joining to help with Helm documentation. With their background, yxxhero will continue to help strengthen both our simplified and traditional Chinese docs.
```

**Review by bridgetkromhout — APPROVED** [review]
LGTM!

**Pull request merged**